Tools/CISupport/ews-build/steps_unittest.py

```diff
@@ -3905,8 +3905,8 @@ def test_success_apple(self):
                 timeout=600,
                 logEnviron=False,
                 env=self.ENV,
-                command=['git', 'cherry-pick', '--allow-empty', 'HEAD..remotes/Contributor-apple/eng/pull-request-branch'],
-            ) + 0,
+                command=['git', 'cherry-pick', '--allow-empty', 'HEAD..remotes/Contributor-apple/eng/pull-request-branch']
```

Comment:
These changes in `TestCheckOutPullRequest` seems completely unrelated to this PR. 
Is this a style fix for something style checker was complaining? If so, might be better to fix all similar issue in a separate PR. There seems to be many other places using similar formatting, e.g.: https://github.com/WebKit/WebKit/blob/main/Tools/CISupport/ews-build/steps_unittest.py#L3527